getting-started/traced-example/package.json

```diff
@@ -9,13 +9,13 @@
   "author": "OpenTelemetry Authors",
   "license": "Apache-2.0",
   "dependencies": {
-    "@opentelemetry/core": "^0.15.0",
-    "@opentelemetry/exporter-zipkin": "^0.15.0",
-    "@opentelemetry/node": "^0.15.0",
+    "@opentelemetry/core": "^0.16.0",
+    "@opentelemetry/exporter-zipkin": "^0.16.0",
+    "@opentelemetry/node": "^0.16.0",
     "@opentelemetry/plugin-express": "^0.12.0",
```

Comment:
I don't think that this example will work as 0.16.0 is not compatible to 0.12.0 plugins

Reply:
hmm you right but current state we have (0.15 with 0.12) also doesn't work

Reply:
but if you pay attention to package-lock.json it will install version `^0.14.0` anyway. I think we should remove dependency from contrib packages from this repo oterwise it will always break

Reply:
Yes, it has not worked before and it will not work now. Even if 0.14.0 is installed within the plugin this ends up in a non working mixture.
Anyhow, it seems app.js is not requiring tracing.js here so it will no crash at least - but it will also not create any spans.
Will create an issue to followup on this.

Reply:
downgraded, and proper fix (moving the express out of core) should be done